### PR TITLE
fix: incorrect date showing on price chart

### PR DESCRIPTION
**Pull request by chxash**
## Description

Fixes https://github.com/shapeshift/web/issues/700

## Notice

Before submitting a pull request, please make sure you have answered the following:

- [x] Have you followed the guidelines in our [Contributing]('https://github.com/shapeshift/web/CONTRIBUTING.md) guide?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/shapeshift/web/pulls) for the same update/change?
- [x] Do all new and existing tests pass? Does the linter pass?

## Pull Request Type

- [x] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [ ] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

If applicable, please link to the github issue and put `closes #XXXX` in your comment to auto-close the issue that your PR fixes.

## Testing

Please outline all testing steps

1. Reproduce bug
2. Find place in code where bug probably lies
3. Find other chart code where there is no bug
4. Compare and fix bug in first place
5. Verify bug is fixed

## Screenshots (if applicable)

fixed

### CODE REVIEW

File: src/state/slices/marketDataSlice/marketDataSlice.tsx [hunk 1]
```diff
@@ -158,7 +158,7 @@ export const selectPriceHistoryByAssetTimeframe = createSelector(
     (priceHistory[timeframe][assetId] ?? []).map(({ price, date }) => ({
       price,
       // TODO(0xdef1cafe): find best primitive to return/store this
-      date: new Date(Number(date) * 1000).toISOString()
+      date: date
     }))
 )
```
Comment (0xdef1cafe) [inline src/state/slices/marketDataSlice/marketDataSlice.tsx:163]: hey @chxash - thanks for the PR. testing this locally it looks like this fixes things, so if that's the case, can you please go ahead and remove this whole `map` function, as it's not actually doing anything any longer?

**Review by 0xdef1cafe — CHANGES_REQUESTED** [review]

File: src/state/slices/marketDataSlice/marketDataSlice.tsx (hunk 1 above)
Comment (chxash) [reply src/state/slices/marketDataSlice/marketDataSlice.tsx]: hi @0xdef1cafe, you are right :) I removed it

**Review by 0xean — APPROVED** [review]

**Comment by chxash** [comment]
@0xean ah wow thank you very much for the tip !
I have already been on the discord and I like the spirit there, is there a specific channel for developers ?
I am interested in contributing more in the future.

**Comment by 0xean** [comment]
Yes ser!

Please select engineer role in #role-selector and feel free to say hello in #engineering-public

**Pull request merged**